src/lang/en-US/AdminPage.ts

```diff
@@ -121,6 +121,17 @@ export const AdminPage: Record<string, string> = {
     'admin.alerts.table.noContent.message': `To begin receiving email notifications for a prefix you admin, click "Configure Notifications" above to create a subscription.`,
     'admin.alerts.table.label.alertMethod': `Notification Method`,
 
+    'admin.dataPlanes.private.header': `Private Data Planes`,
+    'admin.dataPlanes.table.aria.label': `Data Planes Table`,
+    'admin.dataPlanes.table.columns.scope': `Scope`,
+    'admin.dataPlanes.table.columns.name': `Name`,
+    'admin.dataPlanes.table.columns.reactor': `Reactor Address`,
+    'admin.dataPlanes.table.columns.ips': `IP Blocks`,
+    'admin.dataPlanes.table.columns.ips.missing': `None configured`,
+    'admin.dataPlanes.table.noContent.header': `No private data planes found`,
+    'admin.dataPlanes.table.noContent.message': `To being configuring a private data please contact support.`,
```

Comment:
There is a typo in this sentence: _To begin configuring a private data plane, please contact support._ That said, how do you feel about dropping the word _begin_? This would leave you with: _To configure a private data plane, please contact support._